src/components/AboutPage.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import {Link} from 'react-router-dom'
+import Header from './Header'
 import react from './icon/react.svg'
 import tailwind from './icon/tailwind.svg'
 import postgres from './icon/postgres.png'
 
 function AboutPage() {
   return (
     <div className="h-screen w-screen bg-gray-100">
-      <div className={`header`}>Messages</div>
+      <Header>
+        Resourses
```

Comment:
You've got a typo here.

```suggestion
        Resources
```